docs/UserGuide.md

```diff
@@ -51,7 +55,8 @@ HealthSync is a **desktop app for managing patient details, optimised for use vi
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
-* The word `or` indicates that at least one of the fields has to be supplied by the user. The output `or` will return the above field that was supplied by the user.
+* The word `or` indicates that at least one of the fields has to be supplied by the user. 
```

Comment:
Missed out these collasped lines in my first review.

Refer to lines 37. Change `ward/` to `/w` and `phone/` to `p/`.
Refer to lines 41: Should `clear` delete all patients or clear the entry?

For some reason GitHub does not allow commenting on collasped lines in a PR. Refer to [https://github.com/isaacs/github/issues/1655](url)

Reply:
`clear` is a command to delete ALL patients - this facilitates the removal of dummy entries from the system so that real data can be populated into the program, and it is a legacy feature from AddressBook.

Reply:
I see. Then referring to line 232, I think it would be better to rephrase `clear all entries` as `delete all patients` instead of `clear all entries`.